examples/frameworks/strands_demo/src/nat_strands_demo/url_directory.py

```
@@ -0,0 +1,113 @@
+                "   - method: 'GET'",
+                "   - url: '<selected_url>'",
+                "   - convert_to_markdown: true (boolean, NOT string)",
+                "3. Do NOT include any other optional parameters (no auth_type, headers, body, etc.)",
+                "4. Example: http_request(method='GET', url='<selected_url>', convert_to_markdown=true)",
+                "",
+                f"Query context: {query}",
+            ])
+
+            return "\n".join(directory_lines)
+
+        except Exception as e:  # noqa: BLE001
```

Ruff (0.14.4)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)